inttest/calico/calico_test.go

```diff
@@ -58,6 +63,44 @@ func (s *CalicoSuite) TestK0sGetsUp() {
 
 	s.T().Log("waiting to see calico pods ready")
 	s.NoError(common.WaitForDaemonSet(s.Context(), kc, "calico-node"), "calico did not start")
+	s.NoError(common.WaitForPodLogs(s.Context(), kc, "kube-system"))
+
+	createdTargetPod, err := kc.CoreV1().Pods("default").Create(s.Context(), &corev1.Pod{
+		TypeMeta:   metav1.TypeMeta{Kind: "Pod", APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: "nginx"},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{{Name: "nginx", Image: "docker.io/library/nginx:1.23.1-alpine"}},
+			NodeSelector: map[string]string{
+				"kubernetes.io/hostname": "worker0",
+			},
+		},
+	}, metav1.CreateOptions{})
+	s.Require().NoError(err)
+	s.Require().NoError(common.WaitForPod(s.Context(), kc, "nginx", "default"), "nginx pod did not start")
+
+	targetPod, err := kc.CoreV1().Pods(createdTargetPod.Namespace).Get(s.Context(), createdTargetPod.Name, metav1.GetOptions{})
+	s.Require().NoError(err)
+
+	sourcePod, err := kc.CoreV1().Pods("default").Create(s.Context(), &corev1.Pod{
+		TypeMeta:   metav1.TypeMeta{Kind: "Pod", APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: "busybox"},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{{
+				Name:    "busybox",
+				Image:   "gcr.io/google-containers/busybox",
```

Comment:
This image is not available for arm64:

➜  ~ docker run -it --rm  gcr.io/google-containers/busybox bash
<snip>
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

In the kuberouter hairpin test we have the image docker.io/curlimages/curl:7.84.0 which has sleep and curl and is available in arm64

I'm not quite sure if we will ever want to test this on more architectures than amd64, if this is an acceptable constraint I think we can leave it as is.

Reply:
@juanluisvaladas that's a very good point. I think it makes sense to change the image to something which is multiarch. Maybe even just plain `alpine:3.16` as that also has both `sleep` and `wget` available

Reply:
Changed the image to `alpine:3.16`. Thanks!